index.js

```diff
@@ -24,15 +24,15 @@ var routes = require('./app/routes');
 
 var app = express();
 
-var mongoUri = `mongodb://${secrets.db_user}:${secrets.db_password}@${secrets.db_IP}:${secrets.db_port}/${secrets.db}`;
-var mongoSslOpt = {
-    "server": {
-        "sslValidate": false,
-        "sslKey": fs.readFileSync('/etc/ssl/mongodb.pem'),
-        "sslCert": fs.readFileSync('/etc/ssl/mongodb.pem'),
-        }
-    }
+var mongoUri = `mongodb://${config.db_user}:${config.db_password}@${config.db_IP}:${config.db_port}/${config.db}`;
 if (app.get('env') === 'production') {
+    var mongoSslOpt = {
```

Comment:
Had to move this here, because it was breaking dev for me since I don't have these .pem files.